alternator/executor.hh

```diff
@@ -199,13 +202,16 @@ public:
     future<request_return_type> describe_continuous_backups(client_state& client_state, service_permit permit, rjson::value request);
 
     future<> start();
-    future<> stop() { return make_ready_future<>(); }
+    future<> stop() {
+        // disconnect from the value source, but keep the value unchanged.
+        s_default_timeout_in_ms = utils::updateable_value<uint32_t>{s_default_timeout_in_ms()};
+        return make_ready_future<>();
+    }
 
     static sstring table_name(const schema&);
     static db::timeout_clock::time_point default_timeout();
-    static void set_default_timeout(db::timeout_clock::duration timeout);
 private:
-    static db::timeout_clock::duration s_default_timeout;
+    static thread_local utils::updateable_value<uint32_t> s_default_timeout_in_ms;
```

Comment:
Why not make it a regular member, now that it is passed through the constructor?

Reply:
what do you mean by "regular"? i think, if we want to keep `executor::default_timeout()` a static method, we would have to make `s_default_timeout_in_ms` static at least. if we want to avoid possible cross core racings when assigning this static variable, we need to make this variable thread local.

Reply:
Ok, I would say we should get rid of the static `default_timeout()` but that is outside the scope of this patch.